sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+from scipy.sparse import csr_matrix
+from scipy.linalg import block_diag
+from sklearn.decomposition import LatentDirichletAllocation
+
+from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import raises
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_greater_equal
+from sklearn.utils.testing import if_not_mac_os
+
+from sklearn.utils.validation import NotFittedError
+from sklearn.externals.six.moves import xrange
+
+
+def _build_sparse_mtx():
+    """
+    Create 3 topics and each have 3 words
```

Comment:
please use comments and not docstrings in tests, so
# Create 3 topics ..

instead of
``"""``.